web/themes/custom/pineapple/templates/views/views-view--recipes.html.twig

```diff
@@ -30,15 +30,15 @@
  * @see template_preprocess_views_view()
  */
 #}
-{%
-  set classes = [
+{% set classes = [
     'view',
     'view-' ~ id|clean_class,
     'view-id-' ~ id,
     'view-display-id-' ~ display_id,
     dom_id ? 'js-view-dom-id-' ~ dom_id,
   ]
 %}
+{{ attach_library('pineapple/filters') }}
```

Comment:
Attach library should be at the top of the file (before any other code and after comments)

Reply:
Done